card/mod/bootstrap/db/migrate_cards/20170719163710_update_bootswatch_themes_to_4_beta.rb

```diff
@@ -13,8 +13,10 @@ def up
   def update_default_bootstrap
     paths = [Skin.bootstrap_scss_path("functions"),
              Skin.bootstrap_scss_path("variables")]
+    puts "Update bootstrap default"
     Skin.new("Bootstrap default")
         .update_scss field_name: "variables", file_name: paths
+    puts "Finished"
```

Comment:
did you decide to leave these in?

Reply:
yes, deploying hangs somewhere around here